ksqldb-common/src/main/java/io/confluent/ksql/util/KsqlConfig.java

```diff
@@ -827,6 +832,14 @@ private static ConfigDef buildConfigDef(final ConfigGeneration generation) {
                 + "By convention, the id should end in a seperator character of some form, e.g. "
                 + "a dash or underscore, as this makes identifiers easier to read."
         )
+        .define(
+            KSQL_SERVICE_NAME_CONFIG,
+            ConfigDef.Type.STRING,
+            KSQL_SERVICE_NAME_DEFAULT,
+            ConfigDef.Importance.MEDIUM,
+            "Indicates the NAME of the ksql service. It will be used for RBAC "
+                + "authorization instead of the existing service ID above."
```

Comment:
We don't need to specify this is for RBAC purposes. The authorization plugins do not accept the service ID or Name. It just accepts a ksqlConfig object. It's up to the authorization plugin to use an Id or the name for authorization.